frontend/resourceadm/pages/AboutResourcePage/AboutResourcePage.tsx

```diff
@@ -116,14 +116,6 @@ export const AboutResourcePage = ({
     resourceData.delegable ? getResourcePageTextfieldError(resourceData.rightDescription) : false,
   );
 
-  // useRefs to handle tabbing between the input elements and the right translation bar
```

Comment:
Thank you for fixing this! 👏 @framitdavid and myself had a discussion about this a few weeks ago. This has been on my agenda for a few weeks now, so I'm glad to see you have fixed it!